interactive/commands.py

```diff
@@ -76,11 +77,11 @@ def create_user(*, creator, email, name, project):
         for org in project.orgs.all()
     )
 
-    ProjectMembership.objects.create(
-        created_by=creator,
+    members.add(
         project=project,
         user=user,
         roles=[InteractiveReporter],
+        by=creator,
```

Comment:
Is it worth having an assertion on `test_create_user` to ensure the audit log is created?